specs/spec_helper.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../far_mar'
```

Comment:
It's kinda pedantic, but simplecov _demands_ it be the first thing. Take a moment to move this line below `SimpleCov.start`